experimental/navigation/otel/spanCreator.ts

```diff
@@ -10,16 +10,16 @@ const ATTRIBUTES = {
 const spanStart = (
   tracer: TracerRef,
   span: SpanRef,
-  currentRouteName: string,
-  isFirstView: boolean,
+  name: string,
+  isFirstView?: boolean,
```

Comment:
if we're re-using this probably `isFirstView` could be moved out as well and have the caller set the attributes. Another option for keeping this independent would be to duplicate the helpers from navigation into here and then refactor them later if we have a standalone helper package